static/templates/report_message.hbs

```diff
@@ -0,0 +1,50 @@
+<!-- TODO: add docs and link to them -->
+<!-- future: support tabindex / tabbing to help users reporting multiple messages -->
+<div class="reportmsg-box" class="controls" id="message-history" style="display:none">
```

Comment:
Assuming we indeed go with change in the field name, I think these names would become, for example, `report-message-box`.

Reply:
WILLDO - the renaming will include catch any mention of "msg" anywhere in my PR.